### PR TITLE
Pivot to a virtual event

### PR DESCRIPTION
**Pull request by anssiko**

### CODE REVIEW

**Comment by anssiko** [comment]
[HTML preview](https://cdn.statically.io/gh/w3c/machine-learning-workshop/virtual-workshop/index.html)

**Comment by anssiko** [comment]
Pushed an update to reorder lightning talks under four top-level topics. Thanks @kdavis-mozilla!

**Comment by dontcallmedom** [comment]
@anssiko I have added a first [stab of a page for speakers](https://cdn.statically.io/gh/w3c/machine-learning-workshop/virtual-workshop/speakers.html) - let me know what you think

File: speakers.html [hunk 1]
```diff
@@ -0,0 +1,136 @@
...
+        </section>
+        <section>
+          <h2 id=what>What should my talk cover?</h2>
+          <p>The Program Committee will review proposed talks to assess they fit well with the call for participation, in particular the list of <a href="#topics">identified topics</a>.</p>
+          <p>Once selected, speakers should aim to provide a talk that:</p>
+          <ul>
+            <li>brings their specific perspective on the topic,</li>
+            <li>identifies what barriers may need to be lifted to make browsers a great platform for deploying machine learning applications and services, in particular those where <strong>interoperability and standadization</strong> are likely to play a role,</li>
+            <li>shed light on specific aspects or questions the Program Committee will have raised in their review process,</li>
+            <li>raise questions of their own that other Workhsop participants may usefully provide input on.</li>
```
Comment (anssiko) [inline speakers.html:90]: Workshop

File: speakers.html [hunk 2]
```diff
@@ -0,0 +1,136 @@
...
+          <p>Talks are expected to be delivered in English; as an experiment, the Program Committee will also accept talks delivered in Chinese and Japanese, and W3C will work with the speakers on ensuring their proper transcription and translation in English.</p>
+
+          <p>W3C will provide transcripts and captions for all the selected presentations and will ask speakers' help in reviewing these for accuracy.</p>
+        </section>
+        <section>
+          <h2 id=other>What other commitments are expected from me if I give a talk?</h2>
+          <p>A few weeks after all the pre-recorded talks have been published on the W3C site, the Program Committee will set up a series of teleconferences where each of the workshop topics will get discussed, as informed by the submitted presentations.</p>
+          <p>Speakers are expected to the best of their availability to be one the call(s) where the topic of their presentation is being discussed, to answer and raise questions with other workshop participants.</p>
```
Comment (anssiko) [inline speakers.html:109]: on the call(s)

**Comment by dontcallmedom** [comment]
@anssiko I made a few last set of changes for a scheduled release early last week, incl link to a registration form for workshop participants, and an overall timeline. Can you take another look?

**Comment by anssiko** [comment]
@dontcallmedom the remaining of the changes look good to me as well.

For consistency it might make sense to use the exactly same title in the ti.to registration form and the homepage and settle on either of:

W3C Workshop on Web and Machine Learning
Web & Machine Learning Virtual Workshop (where preferably s/&/and/ for SEO)

**Comment by dontcallmedom** [comment]
ti.to updated - merging now

**Pull request merged**